robusta_krr/core/integrations/kubernetes.py

```diff
@@ -20,6 +20,8 @@
 from robusta_krr.core.models.result import ResourceAllocations
 from robusta_krr.utils.configurable import Configurable
 
+from .rollout import RolloutAppsV1Api
```

Comment:
why from .rollout and not from rollout?

Reply:
doesnt work without the dot, but i think i should find a way to make it part of "robusta_krr.core.integrations"

Reply:
I think you can refer to it as robusta_krr.core.integrations.rollout